src/Pagination/SlidingPagination.php

```diff
@@ -94,7 +94,12 @@ public function getQuery(array $additionalQueryParams = []): array
         return \array_merge($this->params, $additionalQueryParams);
     }
 
-    public function isSorted(?string $key = null, array $params = []): bool
+    /**
+     * @param string[]|string|null $key
+     * @param array $params
```

Comment:
`@return null` is unnecessary.
` @param array $params` should be expanded (e.g. to `array|mixed[]`) otherway is useless

Reply:
There is no ```@return null```... that would be incorrect, since it returns a bool. ```@param null``` was effectively already in place, since the ```$key``` parameter was optional.
Why is 'array' useless? In my opinion ```array``` and ```mixed[]``` are synonyms: arrays with unspecified content type

Reply:
Sorry I meant `@return bool`.
Every phpdoc notation that doesn't add information should be avoided. This is part of `no_superfluous_phpdoc_tags` rule of php-cs-fixer (that we're using here as part of `@Symfony` rule)

Reply:
OK, I'll remove the ```@return```! Shouldn't we remove the ```@param array $params``` too, based on this rule?

Reply:
Exactly. Unless more info is provided (like with `@param array|mixed[] $params`)